lib/tests/regex-with-escapes.test.ts

```diff
@@ -8,11 +8,11 @@ test("regex-with-escapes", () => {
             properties: {
                 str: { 
                     type: "string",
-                    pattern: "^\/$"
+                    pattern: "^/$"
```

Comment:
When this string is loaded into memory it will be: `^/$`.

```
console.log("^/$")
^/$
```

The previous `\` was unnecessary, an escape character for a character that doesn't need escaping, resulting in the character anyways:

```
console.log("^\/$")
^/$
```